staging/src/k8s.io/apiserver/pkg/storage/value/transformer.go

```diff
@@ -100,6 +106,10 @@ func (t *MutableTransformer) TransformToStorage(ctx context.Context, data []byte
 	return transformer.TransformToStorage(ctx, data, dataCtx)
 }
 
+func (t *MutableTransformer) Stop() error {
+	return t.transformer.Stop()
```

Comment:
the other operations are protected by a lock, don't we need to use the lock here too?

Reply:
sorry for missed this, already updated.